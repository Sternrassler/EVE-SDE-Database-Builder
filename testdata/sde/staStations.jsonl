{"stationID":60003760,"stationName":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","solarSystemID":30000142,"operationID":null,"stationTypeID":1529,"corporationID":1000035,"dockingCostPerVolume":0.0,"maxShipVolumeDockable":50000000.0,"officeRentalCost":879488,"reprocessingEfficiency":0.5,"reprocessingStationsTake":0.05,"reprocessingHangarFlag":4,"x":2.0e11,"y":1.0e10,"z":-5.0e10}
{"stationID":60008494,"stationName":"Amarr VIII (Oris) - Emperor Family Academy","solarSystemID":30002187,"operationID":null,"stationTypeID":1932,"corporationID":1000084,"dockingCostPerVolume":0.0,"maxShipVolumeDockable":50000000.0,"officeRentalCost":723634,"reprocessingEfficiency":0.5,"reprocessingStationsTake":0.05,"reprocessingHangarFlag":4,"x":1.5e11,"y":2.0e10,"z":1.0e11}
{"stationID":60004588,"stationName":"Rens VI - Moon 8 - Brutor Tribe Treasury","solarSystemID":30002510,"operationID":null,"stationTypeID":1529,"corporationID":1000051,"dockingCostPerVolume":0.0,"maxShipVolumeDockable":50000000.0,"officeRentalCost":654321,"reprocessingEfficiency":0.5,"reprocessingStationsTake":0.05,"reprocessingHangarFlag":4,"x":-1.0e11,"y":3.0e10,"z":-2.0e11}
