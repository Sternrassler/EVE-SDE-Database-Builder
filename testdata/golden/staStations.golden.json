[
  {
    "stationID": 60003760,
    "security": null,
    "dockingCostPerVolume": 0,
    "maxShipVolumeDockable": 50000000,
    "officeRentalCost": 879488,
    "operationID": null,
    "stationTypeID": 1529,
    "corporationID": 1000035,
    "solarSystemID": 30000142,
    "constellationID": null,
    "regionID": null,
    "stationName": "Jita IV - Moon 4 - Caldari Navy Assembly Plant",
    "x": 200000000000,
    "y": 10000000000,
    "z": -50000000000,
    "reprocessingEfficiency": 0.5,
    "reprocessingStationsTake": 0.05,
    "reprocessingHangarFlag": 4
  },
  {
    "stationID": 60008494,
    "security": null,
    "dockingCostPerVolume": 0,
    "maxShipVolumeDockable": 50000000,
    "officeRentalCost": 723634,
    "operationID": null,
    "stationTypeID": 1932,
    "corporationID": 1000084,
    "solarSystemID": 30002187,
    "constellationID": null,
    "regionID": null,
    "stationName": "Amarr VIII (Oris) - Emperor Family Academy",
    "x": 150000000000,
    "y": 20000000000,
    "z": 100000000000,
    "reprocessingEfficiency": 0.5,
    "reprocessingStationsTake": 0.05,
    "reprocessingHangarFlag": 4
  },
  {
    "stationID": 60004588,
    "security": null,
    "dockingCostPerVolume": 0,
    "maxShipVolumeDockable": 50000000,
    "officeRentalCost": 654321,
    "operationID": null,
    "stationTypeID": 1529,
    "corporationID": 1000051,
    "solarSystemID": 30002510,
    "constellationID": null,
    "regionID": null,
    "stationName": "Rens VI - Moon 8 - Brutor Tribe Treasury",
    "x": -100000000000,
    "y": 30000000000,
    "z": -200000000000,
    "reprocessingEfficiency": 0.5,
    "reprocessingStationsTake": 0.05,
    "reprocessingHangarFlag": 4
  }
]